src/main/resources/resources/HomepageResources_pt.properties

```diff
@@ -1,49 +1,49 @@
+link.homepage.listings = P\u00e1ginas Pessoais
 link.homepage.listings.alumni = Alumni
-link.homepage.listings.employees = Funcionários
+link.homepage.listings.employees = Funcion\u00e1rios
 link.homepage.listings.students = Alunos
 link.homepage.listings.teachers = Docentes
-link.homepage.statistics = Estatísticas
-message.homepage.activation = Neste espaço pode activar a sua página pessoal e escolher a informação que deseja incluir. <br/> Se desejar ter uma página pessoal mais flexível e com execução de scripts cgi e php deverá usar o <a target="_blank" href="https://ciist.ist.utl.pt/servicos/web.php">Serviço de WEB</a>. <br/> Os conteúdo a introduzir na página pessoal devem estar de acordo com as <a target="_blank" href="http://dsi.tecnico.ulisboa.pt/normas/">normas de utilização de recursos informáticos do {0}</a>. <br/> Depois de disponível a página fica alojada no endereço indicado.
-message.homepage.options = Caso pretenda actualizar a sua fotografia, poderá fazê-lo na secção <em>Informação</em> ou alternativamente junto do Operador na recepção da DSI.<br/>Se desejar actualizar os seus contactos pessoais poderá fazê-lo na secção <em>Informação</em>.
-message.resource.not.available.for.external.users = Este recurso apenas é disponível para utilizadores da instituição.
-person.homepage.adress = Em caso afirmativo, a página fica alojada em
+link.homepage.statistics = Estat\u00edsticas
+message.homepage.activation = Neste espa\u00e7o pode activar a sua p\u00e1gina pessoal e escolher a informa\u00e7\u00e3o que deseja incluir. <br/> Se desejar ter uma p\u00e1gina pessoal mais flex\u00edvel e com execu\u00e7\u00e3o de scripts cgi e php dever\u00e1 usar o <a target="_blank" href="https://ciist.ist.utl.pt/servicos/web.php">Servi\u00e7o de WEB</a>. <br/> Os conte\u00fado a introduzir na p\u00e1gina pessoal devem estar de acordo com as <a target="_blank" href="http://dsi.tecnico.ulisboa.pt/normas/">normas de utiliza\u00e7\u00e3o de recursos inform\u00e1ticos do {0}</a>. <br/> Depois de dispon\u00edvel a p\u00e1gina fica alojada no endere\u00e7o indicado.
+message.homepage.options = Caso pretenda actualizar a sua fotografia, poder\u00e1 faz\u00ea-lo na sec\u00e7\u00e3o <em>Informa\u00e7\u00e3o</em> ou alternativamente junto do Operador na recep\u00e7\u00e3o da DSI.<br/>Se desejar actualizar os seus contactos pessoais poder\u00e1 faz\u00ea-lo na sec\u00e7\u00e3o <em>Informa\u00e7\u00e3o</em>.
+message.resource.not.available.for.external.users = Este recurso apenas \u00e9 dispon\u00edvel para utilizadores da institui\u00e7\u00e3o.
+person.homepage.adress = Em caso afirmativo, a p\u00e1gina fica alojada em
 person.homepage.submit = Submeter
 person.homepage.update = Actualizar
-person.homepage.url = Endereço
-title.homepage.of = Página Pessoal de
-title.manage.homepage = Gestão da Página Pessoal
+person.homepage.url = Endere\u00e7o
+title.homepage.of = P\u00e1gina Pessoal de
+title.manage.homepage = Gest\u00e3o da P\u00e1gina Pessoal
```

Comment:
Revert this changes. The file should stay the same with accentuated characters, the file encoding should be ISO-8859-1.